platform/harmony/transaction.go

```diff
@@ -64,7 +64,7 @@ func NormalizeTx(trx *Transaction) (tx types.Tx, b bool, err error) {
 
 	return types.Tx{
 		ID:       trx.Hash,
-		Coin:     coin.ONE,
+		Coin:     coin.HARMONY,
```

Comment:
```suggestion
		Coin:     coin.HARMONY,
```

maybe we use `coin.Harmony().ID` for consistency?

Reply:
I'd like keep direct accessing `coin.HARMONY`, unlike `name`, `symbol` it's defined in the map, less call